proof/crefine/AARCH64/Fastpath_C.thy

```diff
@@ -84,16 +81,15 @@ declare comp_apply [simp]
 crunch ksArch[wp]: asUser "\<lambda>s. P (ksArchState s)"
   (wp: crunch_wps)
 
-definition
-   tcbs_of :: "kernel_state => word32 => tcb option"
-where
-  "tcbs_of s = (%x. if tcb_at' x s then projectKO_opt (the (ksPSpace s x)) else None)"
+(* FIXME AARCH64 consider moving this, on MCS there is a tcbs_of as well *)
+definition tcbs_of :: "kernel_state \<Rightarrow> machine_word \<Rightarrow> tcb option" where
+  "tcbs_of s = (\<lambda>x. if tcb_at' x s then projectKO_opt (the (ksPSpace s x)) else None)"
```

Comment:
I had completely forgotten that this was in here. Should indeed consolidate these at some point, but fine to leave here for now, esp since the FIXME makes it findable.